### DISCUSSION TITLE
Can't edit the icons on any collections

### DISCUSSION THREAD

**Discussion by santiagopliego** [outline/outline#3312] [General]
As of this morning, editing icons on collections doesn't work. Reproduce:

1. Highlight a collection on the left sidebar and click the three-dot menu.
2. Click on `Edit...`
3. In the _Edit Collection_ screen, click on the `collection icon`.

At this point, _Edit Collection_ screen closes and I get sent back to the document, unable to change the icon or color.

**Comment by tommoor** [comment]
Confirmed, this seems like a regression – we'll get it fixed up
